src/main/java/com/recursive_pineapple/nuclear_horizons/recipes/ChemicalReactor.java

```diff
@@ -0,0 +1,236 @@
+            .duration(20 * SECONDS)
+            .eut(TierEU.RECIPE_MV)
+            .addTo(UniversalChemical);
+
+        GTValues.RA.stdBuilder()
+            .itemInputs(
+                MaterialsNuclear.URANIUM_233_DIOXIDE.get(OrePrefixes.dust, 3),
+                Materials.Sodium.getDust(4))
+            .fluidInputs(
+                Materials.HydrofluoricAcid.getFluid(4000))
+            .itemOutputs(
+                ItemUtils.getItemStackOfAmountFromOreDict("dustUranium233", 1),
+                ItemUtils.getItemStackOfAmountFromOreDict("dustSodiumFluoride", 4))
+            .fluidOutputs(
+                Materials.Water.getFluid(2000))
+            .duration(20 * SECONDS)
+            .eut(TierEU.RECIPE_MV)
+            .addTo(UniversalChemical);
+
+        GTValues.RA.stdBuilder()
+            .itemInputs(
+                MaterialsNuclear.URANIUM_238_DIOXIDE.get(OrePrefixes.dust, 3),
+                Materials.Sodium.getDust(4))
+            .fluidInputs(
+                Materials.HydrofluoricAcid.getFluid(4000))
+            .itemOutputs(
+                Materials.Uranium.getDust(1),
+                ItemUtils.getItemStackOfAmountFromOreDict("dustSodiumFluoride", 4))
+            .fluidOutputs(
+                Materials.Water.getFluid(2000))
+            .duration(20 * SECONDS)
+            .eut(TierEU.RECIPE_MV)
+            .addTo(UniversalChemical);
+
+        //reduction of plutonium oxides to plutonium metal
+        GTValues.RA.stdBuilder()
+            .itemInputs(
+                MaterialsNuclear.PLUTONIUM_241_DIOXIDE.get(OrePrefixes.dust, 3),
+                Materials.Calcium.getDust(2))
+            .fluidInputs(
+                Materials.HydrofluoricAcid.getFluid(4000))
+            .itemOutputs(
+                Materials.Plutonium241.getDust(1),
+                ItemUtils.getItemStackOfAmountFromOreDict("dustCalciumFluoride", 2))
+            .fluidOutputs(
+                Materials.Water.getFluid(2000))
+            .duration(20 * SECONDS)
+            .eut(TierEU.RECIPE_MV)
+            .addTo(UniversalChemical);
+
+        GTValues.RA.stdBuilder()
+            .itemInputs(
+                MaterialsNuclear.PLUTONIUM_239_DIOXIDE.get(OrePrefixes.dust, 3),
+                Materials.Calcium.getDust(2))
+            .fluidInputs(
+                Materials.HydrofluoricAcid.getFluid(4000))
+            .itemOutputs(
+                Materials.Plutonium.getDust(1),
+                ItemUtils.getItemStackOfAmountFromOreDict("dustCalciumFluoride", 2))
+            .fluidOutputs(
+                Materials.Water.getFluid(2000))
+            .duration(20 * SECONDS)
+            .eut(TierEU.RECIPE_MV)
+            .addTo(UniversalChemical);
+
+        //natural uranium tetrafluoride
+        GTValues.RA.stdBuilder()
+            .itemInputs(
+                GTOreDictUnificator.get(OrePrefixes.dust, Materials.Uraninite, 3))
+            .fluidInputs(Materials.HydrofluoricAcid.getFluid(8000))
```

Comment:
I think this needs to be 16k HF. The stoic on the output dust is 4 HF per dust for 4 dust, meaning it requires 16 F. The extra hydrogen can just be voided, it's basically free.

Reply:
Actually, I think the output is wrong. The uranite input only has 1 mol of uranium but the recipe outputs 4 mol of ?UF4, leading to duplications down the line since the other recipes assume 1 mol of the uranium compounds = 3 uranite / 1 mol uranium.

Reply:
The stoic here was in error. I've corrected this to use 16000 HF, and changed the centrifuging recipes to turn 12000 UF6->4 mol U.